frontend/views/ejs/error.ejs

```diff
@@ -1,3 +1,19 @@
-<h1><%= message %></h1>
-<h2><%= error.status %></h2>
-<pre><%= error.stack %></pre>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../../public/stylesheets/error.css"/>
```

Comment:
The `.css` path is not correct. It should not be relative to the current `.ejs` file. It should be written as an absolute one with `/stylesheets` as the root directory when referencing stylesheets.